src/client/components/molecules/homeSections/second/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext, useEffect, useState } from 'react';
+import ItemList from '../../../atoms/itemList';
+import './style.scss';
+import StateContext from '../../../../utils/stateContext';
+import { Portfolio } from '../../../../utils/Type';
+
+const SecondSection = () => {
+  const { handlers } = useContext(StateContext);
+  const [portfolioItems, setPortfolioItems] = useState<Portfolio[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const response = await handlers.getAllPortfolios();

```

Comment:
this request will get expensive in future. what is the pourpose of this?

if the idea is to show some existing portfolios as examples, better to use fake ones.